### PR TITLE
Add recipe for typechecks

### PR DESCRIPTION
**Pull request by npavlovikj**
Add recipe for typechecks

### CODE REVIEW

File: recipes/typechecks/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package:
+  name: typechecks
+  version: "0.1.0"
+
+source:
+  fn: typechecks-0.1.0.tar.gz
+  url: https://files.pythonhosted.org/packages/62/21/15129201c1f52f6af1e7809e96dce46da4b406c2c07fe9425e92f51edc5c/typechecks-0.1.0.tar.gz
+  md5: 140956d8f5991b9b73769431b95aeff8
```
Comment (synapticarbors) [inline recipes/typechecks/meta.yaml]: use `sha256`

File: recipes/typechecks/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+package:
+  name: typechecks
+  version: "0.1.0"
+
+source:
+  fn: typechecks-0.1.0.tar.gz
+  url: https://files.pythonhosted.org/packages/62/21/15129201c1f52f6af1e7809e96dce46da4b406c2c07fe9425e92f51edc5c/typechecks-0.1.0.tar.gz
```
Comment (synapticarbors) [inline recipes/typechecks/meta.yaml]: Use canonical url:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L17

File: recipes/typechecks/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  md5: 140956d8f5991b9b73769431b95aeff8
+
+build:
+  number: 0
+  skip: True  # [win]
```
Comment (synapticarbors) [inline recipes/typechecks/meta.yaml]: Is there a reason to skip windows? There doesn't appear to be any reference to it not being supported that I can find.

File: recipes/typechecks/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+package:
+  name: typechecks
+  version: "0.1.0"
+
+source:
+  fn: typechecks-0.1.0.tar.gz
+  url: https://files.pythonhosted.org/packages/62/21/15129201c1f52f6af1e7809e96dce46da4b406c2c07fe9425e92f51edc5c/typechecks-0.1.0.tar.gz
+  md5: 140956d8f5991b9b73769431b95aeff8
+
+build:
```
Comment (synapticarbors) [inline recipes/typechecks/meta.yaml]: Build using pip:

https://conda-forge.org/docs/meta.html#use-pip

File: recipes/typechecks/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - typechecks
+
+about:
+  home: https://github.com/openvax/typechecks
+  license: Apache
```
Comment (synapticarbors) [inline recipes/typechecks/meta.yaml]: `license: Apache-2.0`

Also please manually include the `license_file`:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

File: recipes/typechecks/build.sh [hunk 6]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```
Comment (synapticarbors) [inline recipes/typechecks/build.sh]: Drop the `build.sh` and use pip:

https://conda-forge.org/docs/meta.html#use-pip

File: recipes/typechecks/meta.yaml (hunk 4 above)
Comment (synapticarbors) [inline recipes/typechecks/meta.yaml]: Consider making `noarch: python`

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

**Comment by dougalsutherland** [comment]
FYI, we're having issues with converting recipes into feedstocks right now (#5846), so it'll take a bit for this recipe to appear.

**Pull request merged**